content/guides/trash/permanently-delete-a-file.md

```diff
@@ -0,0 +1,19 @@
+---
+rank: 2
+related_endpoints:
+ - delete_files_id
```

Comment:
Shouldn't this be `delete_files_id_trash`?

Reply:
I think both are related here so I'll just add a second related endpoint.